sdk/servicebus/service-bus/test/public/sendAndSchedule.spec.ts

```diff
@@ -65,6 +65,7 @@ describe("Sender Tests", () => {
     should.equal(Array.isArray(msgs), true, "`ReceivedMessages` is not an array");
     should.equal(msgs.length, 1, "Unexpected number of messages");
     should.equal(msgs[0].deliveryCount, 0, "DeliveryCount is different than expected");
+    should.equal(msgs[0].state, ServiceBusMessageState.active);
```

Comment:
Currently failing. Some messages come back without the property. Service team is looking into it.

Reply:
Some messages don't have the property because of service side optimization.  If absent it means the message is Active.